in.go

```diff
@@ -109,9 +128,10 @@ func Get(request GetRequest, github Github, git Git, outputDir string) (*GetResp
 
 // GetParameters ...
 type GetParameters struct {
-	SkipDownload    bool   `json:"skip_download"`
-	IntegrationTool string `json:"integration_tool"`
-	GitDepth        int    `json:"git_depth"`
+	SkipDownload            bool   `json:"skip_download"`
+	IntegrationTool         string `json:"integration_tool"`
+	GitDepth                int    `json:"git_depth"`
+	GenerateChangedFileList bool   `json:"list_changed_files"`
```

Comment:
The struct field should mirror the JSON key name, so please rename to `ListChangedFiles`

Reply:
done